src/electron/common/components/collapsible-component-cards-electron.tsx

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
this file should probably be under `src\electron\views\automated-checks\components\`